py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/batcher.py

```diff
@@ -77,6 +75,70 @@ def event_size_over_limit(self) -> bool:
         return False
 
 
+def compress_filepaths(filepaths: List[str]) -> Dict:
```

Comment:
new compression algo.

i wanted to stay human readable for local tests.

it breaks common prefixes and suffixes off and keeps a list of uniques that are in the middle.